tcmalloc/internal/percpu_rseq_aarch64.S

```diff
@@ -154,6 +169,7 @@ label##_trampoline:                                               \
  * we can not guarantee it will we must save and restore the registers used to
  * store the arguments of our functions. The function with most arguments has 5
  * arguments, so we save x0-x4 and lr.
+ * TODO: Add PAC support because we are spiling LR.
```

Comment:
How is PAC support related to BTI here? I'm not sure how this TODO relates to the rest of the patch, so asking to learn here.

Reply:
PAC(Pointer Authentication) is not directly related to BTI. They are however often used together as PAC is meant to protect against ROP attacks and BTI against JOP attacks. Having said that, this was intended as a helpful reminder that in this particular case, because we potentially spill LR we might want to protected it using PAC.